ovn-tester/ovn_workload.py

```diff
@@ -759,18 +760,22 @@ def create_load_balancer(self):
         self.load_balancer = lb.OvnLoadBalancer(f'lb_{self.name}', self.nbctl)
 
     @ovn_stats.timeit
-    def provision_vips_to_load_balancers(self, backend_lists):
-        vip_net = DEFAULT_NS_VIP_SUBNET.next(self.cluster.n_ns)
+    def provision_vips_to_load_balancers(self, backend_lists, version):
+        vip_ns_subnet = DEFAULT_NS_VIP_SUBNET
+        if version == 6:
+            vip_ns_subnet = DEFAULT_NS_VIP_SUBNET6
+        vip_net = vip_ns_subnet.next(self.cluster.n_ns)
         n_vips = len(self.load_balancer.vips.keys())
         vip_ip = vip_net.ip.__add__(n_vips + 1)
 
-        vips = {
-            f'{vip_ip + i}:{DEFAULT_VIP_PORT}': [
-                f'{p.ip}:{DEFAULT_BACKEND_PORT}' for p in ports
-            ]
-            for i, ports in enumerate(backend_lists)
-        }
-        self.load_balancer.add_vips(vips)
+        for i, ports in enumerate(backend_lists):
+            self.load_balancer.add_vip(
+                vip_ip + i,
+                DEFAULT_VIP_PORT,
+                ports,
+                DEFAULT_BACKEND_PORT,
+                version,
+            )
```

Comment:
IIUC, this will result in 3 separate database transactions instead of 1, as it was before.
Is it possible to keep the old behavior?

Reply:
@igsilya ack, I will fix it